modules/replicator/src/testFixtures/java/org/apache/ignite/internal/replicator/ReplicaTestUtils.java

```diff
@@ -37,29 +39,31 @@ public final class ReplicaTestUtils {
      * Returns raft-client if exists.
      *
      * @param node Ignite node that hosts the raft-client.
-     * @param tableId Desired table's ID.
+     * @param tableOrZoneId Desired table or zone ID.
      * @param partId Desired partition's ID.
      *
      * @return Optional with raft-client if exists on the node by given identifiers.
      */
     @TestOnly
-    public static Optional<RaftGroupService> getRaftClient(Ignite node, int tableId, int partId) {
-        return getRaftClient(getReplicaManager(node), tableId, partId);
+    // TODO https://issues.apache.org/jira/browse/IGNITE-22522 tableOrZoneId -> zoneId
+    public static Optional<RaftGroupService> getRaftClient(Ignite node, int tableOrZoneId, int partId) {
+        return getRaftClient(getReplicaManager(node), tableOrZoneId, partId);
     }
 
     /**
      * Returns raft-client if exists.
      *
      * @param replicaManager Ignite node's replica manager with replica that should contains a raft client.
-     * @param tableId Desired table's ID.
+     * @param tableOrZoneId Desired table or zone ID.
      * @param partId Desired partition's ID.
      *
      * @return Optional with raft-client if exists on the node by given identifiers.
      */
     @TestOnly
-    public static Optional<RaftGroupService> getRaftClient(ReplicaManager replicaManager, int tableId, int partId) {
+    // TODO https://issues.apache.org/jira/browse/IGNITE-22522 tableOrZoneId -> zoneId
+    public static Optional<RaftGroupService> getRaftClient(ReplicaManager replicaManager, int tableOrZoneId, int partId) {
```

Comment:
In an earlier change, I added `getZoneRaftClient()` leaving `getRaftClient()` table-oriented, so the caller would decide which method to call. You chose a different approach by modifying `getRaftClient()` and making it universal. Could you please also replace `getZoneRaftClient()` usages with the universal `getRaftClient()`?

Reply:
Sure.